### PR TITLE
#145  App info section content

### PR DESCRIPTION
**Pull request by aggarwalsshreya**
I have done most of the work, but I'm still having trouble organizing the image and the gradient line.  If someone could look over my code that would be helpful.

### CODE REVIEW

**Comment by Hanna-N9** [comment]
I played with your codes. Here is what happened:
- Move LinearGradient component below the first image. 
- Add the circle logo in the that same div as the LinearGradient component
- Remove everything in className of div for LinearGradient 
- But add flex justify-center in className of div to position circle logo (where LinearGradient component is at)
- I put percentage for LinearGradient instead of px since you are using h-screen in app-info className.
- I put absolute and the top in negative as well as size of image for circle logo

Here are the codes (Lines 75-82)
A3915A34-7E99-4501-8415-366849F913A4_4_5005_c

`<div className="flex justify-center">
              Circle logo
              <LinearGradient height="200%" />
            </div>`

**Comment by aggarwalsshreya** [comment]
Thanks Hanna for your help! The screen looks like this now 
image

File: client/src/App.js [hunk 1]
```diff
@@ -96,9 +97,41 @@ function App() {
...
-        <section className="app-info"></section>
-        <section className="lightbox w-full justify-center items-center flex">
+         </section>
+        <section className="app-info w-screen h-screen p-8 relative">
+          <div className="halogen-line absolute bottom-0 left-0"></div>
+          <div className="flex h-1/2 w-full">
+               <div className="w-1/2 flex justify-center">
+                 <img 
+                 className="h-full p-8"
+                 src="https://user-images.githubusercontent.com/13723156/127782437-7f221d77-d4d9-4371-ad6d-ede848f03248.png" alt="" />
```
Comment (mikelbrierly) [inline client/src/App.js:107]: Let's make sure to add `alt` text here describing the content of the images. Otherwise looks great!!!

Comment (aggarwalsshreya) [reply client/src/App.js]: ok, I made the changes. Ready to merge!

**Review by mikelbrierly — APPROVED** [review]

**Pull request merged**